### DISCUSSION TITLE
umi4升级之后，使用theme更改antd基本样式，但是本地启动发现会被原有antd样式覆盖，优先级更低

### DISCUSSION THREAD

**Discussion by yll0910** [umijs/umi#12552] [Q&A]
antd版本是4.x，umrc.ts中是这样配置theme的：并且关闭了mfsu
![image](https://github.com/umijs/umi/assets/44368783/2cae5afd-607a-4bde-ba3d-4c401bbc7871)

![111](https://github.com/umijs/umi/assets/44368783/8d8227bd-49d1-4b0f-b7d3-ba24085e1102)

**Comment by fz6m** [comment]
没遇到你的这个问题，我新建了一个最新版本的 umi 项目，配置上 antd 插件，安装 antd 4 后，使用如下配置：

```ts
// .umirc.ts

  theme: { '@primary-color': '#1DA57A' },
  mfsu: false,
  antd: {},
```

启动项目之后颜色是正常的（绿色）。

如有问题，需要给一个最小复现。
